shared/actions/teams.tsx

```diff
@@ -1289,6 +1289,29 @@ function* teamBuildingSaga() {
   yield* Saga.chainAction(TeamsGen.addedToTeam, closeTeamBuilderOrSetError)
 }
 
+async function showTeamByName(action: TeamsGen.ShowTeamByNamePayload, logger: Saga.SagaLogger) {
+  const {teamName, initialTab, addMembers} = action.payload
+  let teamID: string
+  try {
+    teamID = await RPCTypes.teamsGetTeamIDRpcPromise({teamName})
+  } catch (e) {
+    logger.info(`showTeamByName: team "${teamName}" cannot be loaded: ${e.toString()}`)
```

Comment:
probably worth `warn`

Reply:
This also happens when you get a link for a team you don't have an access to (or it doesn't exist). can also happen if you click on something from an e-mail, but you are logged in as someone else. So it's not necessarily an exceptional situation.

Reply:
Ok, fair.